src/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -68,11 +68,24 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
         <SeoDataForArtwork artwork={artwork} />
 
         {this.renderImageMetaTags()}
+        {this.maybeRenderNoIndex()}
```

Comment:
non-blocking: The `maybe` prefix seems a bit odd and an antipattern. Maybe we can simplify this altogether with something like this?
```
render() {
    const addNoIndex = artwork?.visibilityLevel == "UNLISTED"
    ...
    return (
        ...
        {addNoIndex && <Meta name="robots" content="noindex, follow" />}
    )
}
```

Reply:
Sounds great, will add!